documentation/index.rst

```diff
@@ -1,8 +1,8 @@
 Welcome to KernelWorkflow's Documentation!
 ==========================================
 
-This set of scripts have a simple mission: reduces the environment and setup
-overhead for developing for GNU/Linux. Kw is composed of different scripts
+This set of scripts has a simple mission: reduce the environment and setup
+overhead of developing for GNU/Linux. Kw is composed of different scripts
```

Comment:
I want to remove phrases like `set of scripts` because this does not reflect the kw reality anymore. Rewrite it to something like "kw has a simple mission:"